### PR TITLE
Adding TechRocks Summit 7-8 dec 

### PR DESCRIPTION
**Pull request by pensarguet**
Adding TechRocks Summit 7-8 dec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,6 +278,7 @@ All the data (past and coming) is available publicly in JSON:
 * 7: [TestJS Summit](https://testjssummit.com/) - Berlin (Germany)
 * 7-8: [SymfonyCon Brussels](https://live.symfony.com/2023-brussels-con/) - Brussels (Belgium)
 * 7-8: [DevTernity](https://devternity.com/) - Online
+* 7-8: [TechRocks Summit](https://events.tech.rocks/summit-2023/fr/registration/ticketing) - Comet Meetings - Bourse 75002 Paris (France)
```
Comment (scraly) [inline README.md:281]:
```suggestion
* 7-8: [TechRocks Summit](https://events.tech.rocks/summit-2023/fr/) - Paris (France)
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks
just a suggestion ;)

File: README.md (hunk 1 above)
Comment (pensarguet) [reply README.md:281]: ok ;)

**Comment by scraly** [comment]
thanks :)

**Pull request merged**